web_tree_image_tooltip/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,7 @@
+=================================================
+Display images and icons in tree view via Tooltip
+=================================================
+
+This module defines a images and icons in tree view via Tooltip.
+Optionally, Default width is 30px.
```

Comment:
I don't understand what this means. Do you mean 'Additionally, the default width is 30px'?

Reply:
> Do you mean 'Additionally, the default width is 30px'?

@tarteo Yes, We Added a Image size.

Reply:
Can you try to clear up the description, please. I know what the module does, but I think for somebody that sees it in the app store it's not very clear.

Reply:
@tarteo Done, Added a description like this `This module defines images and icons in tree view via Tooltip. Additionally, Default width with 30px.`
If any doubt, then add your comment/description, So i will re-write that.